src/Extensions/AzureBlobStorage/Program.cs

```diff
@@ -14,8 +12,18 @@ internal class Program
     {
         private static Stream StdInStream = null;
         private static CancellationTokenSource CancelSource = new CancellationTokenSource();
+
+        protected static ILogger Logger { get; set; }
```

Comment:
@jander-msft , I know you left a comment about this on the original PR, but I left it as is for now since it's also being used for logging in `BuildOptions` - let me know how you want it and I can change it.

Reply:
This is okay for now.

In the long term (before merging back to `main`), I would like to see this rewritten using ServiceCollection/ServiceProvider (see https://github.com/dotnet/dotnet-monitor/blob/main/src/Tests/Microsoft.Diagnostics.Monitoring.UnitTestApp/ScenarioHelpers.cs#L29 as an example) and have the main method register the necessary services, pull out the egress provider and invoke it.